src/Hangfire.Mongo/CountersAggregator.cs

```diff
@@ -16,7 +16,7 @@ namespace Hangfire.Mongo
     /// </summary>
     public class CountersAggregator : IBackgroundProcess, IServerComponent
     {
-        private static readonly ILog Logger = LogProvider.GetCurrentClassLogger();
+        private static readonly ILog Logger = LogProvider.For<CountersAggregator>();
```

Comment:
Yes, there's no `GetCurrentClassLogger()` method in .netstandard version of Hangfire